src/PDE/CompFlowProblem/TaylorGreen.h

```diff
@@ -0,0 +1,302 @@
+// *****************************************************************************
+/*!
+  \file      src/PDE/CompFlowProblem/TaylorGreen.h
+  \author    J. Bakosi
```

Comment:
Aren't we removing these to follow a more traditional open source development model?